namespaces/cloud-platform-live-0.k8s.integration.dsd.io/family-mediators-api-staging/resources/ecr.tf

```diff
@@ -0,0 +1,31 @@
+#####################################
+# Family Mediators API ECR repository
+#####################################
+
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "${var.aws_region}"
+}
+
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=1.0"
```

Comment:
ref=2.0 please, see https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases

Reply:
I knew about the new release, but I had the doubt if I don't require the new functionality in release 2.0, could I continue using release 1.0?